frontend/pages/shared-components/info-tooltip/info-tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faQuestionCircle } from '@fortawesome/free-regular-svg-icons'
+import React from 'react' 
+import styles from './info-tooltip.module.css'
+import { tooltipContent, TooltipTypes } from '../../../models'
+
+interface InfoTooltipProps { type: TooltipTypes }
```

Comment:
Can we put in a TODO here to factor out the tooltip container from the content somehow? I ask since the basic info tooltip is great but what if we have another more customized tooltip that displays more than basic info? instead of the `tooltipConent` import can we pass that as a prop from the parent and just display the `{ children }` that way we can reuse the base Tooltip. Does this make sense? can definitely do in another PR but should add at least a TODO.

Reply:
Hey! Yeah I have a tooltip-related TODO in the related model's file - happy to add detail about this suggestion, but feels like it might be out of scope to tackle it in this ticket since ViewerRegistration doesn't utilize that (& as you pointed out, this ticket is already fairly large 😅)

Reply:
Oh, wait! This is already dynamically rendering content based on the "type" prop - so whether we select the content in the parent & pass it to the child, or pass the selector to select it in the child, seems rather inconsequential?

Personally, I like passing a selector (ie `type`) so that the parent has a concise label for the content, which makes reading the parent template code a bit clearer
Because of that, I'm inclined to keep it - but happy to keep discussing!